cmake/SetupSpheral.cmake

```diff
@@ -77,6 +77,7 @@ if(ENABLE_OPENMP)
   list(APPEND SPHERAL_CXX_DEPENDS openmp)
 endif()
 
+#if(ENABLE_CUDA AND NOT SPHERAL_HIDE_GPU)
```

Comment:
Should this just be removed?

Reply:
If it's commented out and you dont need it right now then we can remove it. I was fine with this change as a temporary solution until we roll out the `SPHERAL_*` prefix changes that are coming. So either delete the comment or return it to using this logic.

Reply:
Ok, I've removed it.